### PR TITLE
Fix add style ztl

### PR DESCRIPTION
**Pull request by zhangtianli2006**

### CODE REVIEW

File: src/assets/css/theme.css [hunk 1]
```diff
@@ -54,64 +54,23 @@
...
-    padding: 10px;
+.hljs-variable,
+.hljs-template-variable,
+.hljs-tag,
+.hljs-name,
+.hljs-selector-id,
+.hljs-selector-class,
+.hljs-regexp,
+.hljs-deletion {
+    color: #c44344 !important;
```
Comment (szdytom) [inline src/assets/css/theme.css:75]: don't change origin color!!!

File: src/components/lib/MarkdownContainer.vue [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default {
   },
   mounted() {
     let content = this.content;
-    content = content.replaceAll('\\', '\\\\'); // for KaTeX
+    content = content.replaceAll('\\', '\\\\');
```
Comment (szdytom) [inline src/components/lib/MarkdownContainer.vue:40]: Why you delete this comment?

File: src/assets/css/basic.css [hunk 3]
```diff
@@ -1,5 +1,3 @@
-/* --scrollbar-- */
-
```
Comment (szdytom) [inline src/assets/css/basic.css:2]: Why you delete this comment?

File: src/assets/css/basic.css [hunk 4]
```diff
@@ -18,8 +16,6 @@
     background-color: #808080;
 }
 
-/* --selection-- */
-
```
Comment (szdytom) [inline src/assets/css/basic.css:22]: Why you delete this comment?

File: src/components/user/login.vue [hunk 5]
```diff
@@ -43,18 +43,31 @@ export default {
...
+              message: 'Requesting too frequently',
+              type: 'error'
+            });
           } else {
-            this.$message.error('Unkown error');
+            this.$message({
+              showClose: true,
+              message: 'Unknown error',
+              type: 'error'
+            });
```
Comment (szdytom) [inline src/components/user/login.vue:70]: You should write a function

File: src/components/user/logout.vue [hunk 6]
```diff
@@ -14,18 +14,31 @@ export default {
...
+              message: 'Requesting too frequently',
+              type: 'error'
+            });
           } else {
-            this.$message.error('Unkown error');
+            this.$message({
+              showClose: true,
+              message: 'Unknown error',
+              type: 'error'
+            });
```
Comment (szdytom) [inline src/components/user/logout.vue:41]: You should write a function

File: src/components/user/register.vue [hunk 7]
```diff
@@ -60,19 +61,39 @@ export default {
...
+              });
             }
           });
       } else {
-        this.$message.error('Password mismatch');
+        this.$message({
+          showClose: true,
+          message: 'Password mismatch',
+          type: 'error'
+        });
```
Comment (szdytom) [inline src/components/user/register.vue:96]: You should write a function

File: src/router.js [hunk 8]
```diff
@@ -6,20 +6,25 @@ let router = new Router({
...
+      import ('./components/problem/list.vue')
   }, {
     path: '/problem/:id',
-    component: () => import('./components/problem/view.vue')
+    component: () =>
+      import ('./components/problem/content.vue')
   }, {
     path: '*',
-    component: () => import('./components/404.vue')
+    component: () =>
```
Comment (szdytom) [inline src/router.js:25]: I think origin code style in more clear.

File: src/assets/css/basic.css [hunk 9]
```diff
@@ -35,6 +31,105 @@
     opacity: 0.2;
 }
 
-a {
-    text-decoration: none;
+pre {
```
[the remark this answers is not in the document]
Comment (zhangtianli2006) [reply src/assets/css/basic.css:34]: Actually auto wrap of `pre` should be load everywhere, but OK, I'll fix it

File: src/assets/css/basic.css (hunk 3 above)
Comment (zhangtianli2006) [reply src/assets/css/basic.css:2]: These two were useless.

File: src/router.js (hunk 8 above)
Comment (zhangtianli2006) [reply src/router.js:25]: That's not me

**Pull request merged**